modules/images/image-loading-optimization/detection/detect.js

```diff
@@ -5,6 +5,43 @@ const doc = win.document;
 
 const consoleLogPrefix = '[Image Loading Optimization]';
 
+const storageLockTimeSessionKey = 'iloStorageLockTime';
+
+/**
+ * Checks whether storage is locked.
+ *
+ * @param {number} currentTime    - Current time in milliseconds.
```

Comment:
the `-` on this line and the next seem unintentional?

Reply:
It's intentional, actually:

> You can add a hyphen before the description to make it more readable. Be sure to include a space before and after the hyphen.

https://jsdoc.app/tags-param

Reply:
Ah, I guess I hadn't seen it used before so it looked like a typo 🤷🏼